.husky/post-merge

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
This file is very good news

Reply:
:+1: wouldn't want to waste anybody's time ;)